src/test/java/guitests/guihandles/PersonListPanelHandle.java

```diff
@@ -394,6 +395,20 @@ public boolean isListMatching(int startPosition, Person... persons) {
         return true;
     }
 
+    public boolean isExactList(Person... persons) {
```

Comment:
Will use your implementation when your PR is merged. Will use this in the meantime as a workaround.